sqs/provider.go

```diff
@@ -0,0 +1,444 @@
+package sqs
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"strings"
+	"time"
+
+	"code.cloudfoundry.org/lager"
+	provideriface "github.com/alphagov/paas-service-broker-base/provider"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/cloudformation"
+	"github.com/pivotal-cf/brokerapi"
+	"github.com/pivotal-cf/brokerapi/domain"
+)
+
+var (
+	// capabilities required by cloudformation
+	capabilities = []*string{
+		aws.String("CAPABILITY_NAMED_IAM"),
+	}
+	// NoExistErrMatch is a string to match if stack does not exist
+	NoExistErrMatch = "does not exist"
+	// ErrStackNotFound returned when stack does not exist, or has been deleted
+	ErrStackNotFound      = fmt.Errorf("cloudformation stack does not exist")
+	ErrUpdateNotSupported = errors.New("Updating the SQS queue is currently not supported")
```

Comment:
two different ways of making an error on the above two lines?

Reply:
think they just arn't named well ... one is the "string that we need to match against from aws" and the other is "a go error to return"